src/client/game/index.js

```diff
@@ -37,7 +37,7 @@ export default class Game {
     this.on('playerScored', this.playerScored.bind(this));
     this.on('bonusSpawned', this.field.spawnBonus.bind(this.field));
     this.on('bonusCollected', (bonusesPaddles) => {
-      this.field.setBonuses(bonusesPaddles.bonuses);
+      this.field.setBonuses(bonusesPaddles);
```

Comment:
Since you're no longer extracting a field from bonusPaddles, you can use

```js
this.on('bonusCollected', this.field.setBonuses.bind(this.field))
```

for consistency